### ISSUE TITLE
Reintroduce the `setDisableOnClick` method to `Button` class in Flow

### ISSUE THREAD

**Issue by basil-bourque** [vaadin/flow#4650]
Vaadin 8 offers a [`Button::setDisableOnClick`](https://vaadin.com/api/framework/8.5.2/com/vaadin/ui/Button.html#setDisableOnClick-boolean-) but Vaadin Flow [`Button`](https://vaadin.com/api/platform/11.0.1/com/vaadin/flow/component/button/Button.html) does not.

[Jouni suggests](https://stackoverflow.com/a/52548886/642706) on this Stack Overflow Question, [*Vaadin disable button during an action*](https://stackoverflow.com/q/52546381/642706), that this feature be reintroduced to Flow.

I agree, this pattern of disabling a button while waiting on an async background task seems common in many business/enterprise apps. This feature makes the pattern a bit easier.

**Comment by pleku** [comment]
Duplicate of https://github.com/vaadin/vaadin-button-flow/issues/27
